angular/projects/admin-nrpti/src/app/records/orders/order-add-edit/order-add-edit.component.html

```diff
@@ -161,38 +150,18 @@ <h4>Outcome Details</h4>
         </div>
         <div class="label-pair med">
           <label for="outcomeDescription">Outcome Description</label>
-          <textarea
-            name="outcomeDescription"
-            id="outcomeDescription"
-            type="text"
-            formControlName="outcomeDescription"
-            class="form-control"
-          ></textarea>
+          <textarea name="outcomeDescription" id="outcomeDescription" type="text" formControlName="outcomeDescription"
+            class="form-control"></textarea>
         </div>
       </div>
     </section>
 
-    <section>
+    <section *ngIf="currentRecord?.documents">
```

Comment:
Is `?.` supported?

I thought it wasn't released yet?

Reply:
It should be supported.

Reply:
It's only supported in tsc 3.7, our package.json is referencing ~3.2

Reply:
Looks like its only supported in a select few browser versions: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining

Probably best to just use `&&` in the meantime